ner_v2/api.py

```diff
@@ -397,6 +397,7 @@ def number_range(request):
     return HttpResponse(json.dumps({'data': entity_output}), content_type='application/json')
 
 
+@csrf_exempt
```

Comment:
Update docs, Message can be list or str.  Also response will be list of list for bulk call.